docs/installing-northstar/using-northstar/playing-on-linux.md

```diff
@@ -155,15 +69,11 @@ Running the game on fullscreen through Linux might lead to a black screen preven
 
 For more info and proposed fixes, refer to [this issue thread on Github](https://github.com/R2Northstar/Northstar/issues/1)
 
-### LatencyFleX issues
-
-Some users have reported issues with enabling LatencyFleX. If you see `"Unable to load LatencyFleX library, LatencyFleX disabled."` in your logs, try adding `latencyflex_wine.dll` to your `bin/x64_retail` folder.
-
 ### Game crashes on launch with Cause: Access Violation Data Execution Prevention (DEP) at: 0x00000000
 
-Try running with [ProtonGE](https://github.com/GloriousEggroll/proton-ge-custom/). In particular, `Proton-7.3-GE-1` (not `GE-Proton7-3`) has so far been found to be reliable in this case. Check this [Github issue comment for a guide](https://github.com/R2Northstar/Northstar/issues/1#issuecomment-1062483190).
+For Steam and Steam Deck, run Northstar with the latest release of [NorthstarProton](https://github.com/cyrv6737/NorthstarProton/releases/latest).
```

Comment:
While I prefer just supporting Steam version of Titanfall2 we should still leave info here for those having same issue on Origin+Lutris or Origin+Bottles, etc.

Reply:
I'll restore this when I redo the stutter section

Reply:
Addressed in latest commit.